wattwatchers_rest_api_v3_long_energy.ipynb

```diff
@@ -74,6 +74,11 @@
     "                                      # If True, saves data for each device to its own CSV file; the file name will include the device id. \n",
     "                                      # If False, saves all device data to a single CSV file and outputs a second file allowing individual device channels to be mapped to their categories\n",
     "\n",
+    "DEVICES = []                          # List of device ids. \n",
+    "                                      # If left as en empty list ([]) Long Energy data for all devices  associated with the API key is downloaded.\n",
```

Comment:
Typo `an`

Reply:
Perhaps use backticks for the code reference, e.g. "... list `[]`, Long Energy..."

Reply:
Extra space "all devices associated"